code/modules/mapping/reader.dm

```diff
@@ -302,6 +302,9 @@
 	//Instanciation
 	////////////////
 
+	//turn off base new Initialization until the whole thing is loaded
+	SSatoms.map_loader_begin()
```

Comment:
This changes functionality pretty seriously for anything loaded later.

Reply:
[`instance = new atype(null)`](https://github.com/tgstation/tgstation/pull/40737/files#diff-b1e23caecb54fba441a5836361af9cf0R318) breaking the area loading.

Reply:
how so?

Reply:
It create area in null space and initialize it before it get a turfs. That produce #40602 problem.
Also it can create problem with markers(or something else) what affects area if area wants it on init.

Reply:
We need init area after installing all parts of area, in any way.